README.md

```diff
@@ -51,6 +51,55 @@ Use these steps if you're working on the Event UI and do not need to run a local
 npm test
 ```
 
+## Testing Practices
+
+Best practice:
+
+For bugfixes, write a failing test before you change the code.
+
+For feature work we expect code to be reasonably well tested, which means:
+
+- A happy path through the feature should be illustrated.
+- Any exception or error handling should be tested.
+- failbacks for sad paths should be tested as well.
+- Features which handle user input should have validation tests describing when the input is valid or invalid.
+  - We would like to have a canonical list of bad user input and are considering fuzzing as well.
+- Tagging tests??? (like @blinks indicates a test intermittently fails.)
+- Should we move to XUnit?
+
+BDD style is being adopted in the codebase, but it was introduced in stages, so many tests aren't using this approach.
+
+- test messages should be human-readable and accurately describe the test. Use `describe` and `context` blocks to establish the context of the method under test, and `it` blocks to describe expectations. Avoid writing more than one `expect` per `it` block.
+  - `describe` should be used to name the object under test
+  - `context` should be used to enumerate the conditions in the test, and begin with 'when'.
+  - `it` blocks should begin with 'should' and list the expectation.
+- When submitting a pull request, try to include a user story ilustrating the happy path of your feature.
+- Any changes to the UI should include before and after screenshots. Or gifs!!
+- In code review, avoid approving if you are not confident in your understanding of the code. Approve after you've pulled the change into your local environment and manually validate.
```

Comment:
If another team member cannot verify that the work is correct by manually testing it in their development environment, the pull requestor should update the User Story associated with the work to clarify and document the intended behavior.